src/plugins/unified_field_list/public/components/field_list/field_list_grouped.tsx

```diff
@@ -60,6 +62,14 @@ function InnerFieldListGrouped<T extends FieldListItem = DataViewField>({
     )
   );
 
+  useEffect(() => {
+    // Reset the scroll if we have made material changes to the field list
```

Comment:
material like ... significant, right .. not like changing from wood to metal? ... ok .. it's late an my brain was just giving me german translations ... pls ignore!

Reply:
@kertal It was in Len code base originally. Decided to keep :)